tests/expr_and_series/over_test.py

```diff
@@ -95,11 +95,6 @@ def test_over_multiple(constructor: Constructor) -> None:
         "i": list(range(5)),
         "c_min": [5, 4, 1, 1, 2],
     }
-    expected = {
-        "a": ["a", "a", "b", "b", "b"],
-        "b": [1, 2, 3, 5, 3],
-        "c": [5, 4, 3, 2, 1],
-    }
```

Comment:
here there were 2 `expected`, the latest of which didn't have the most important column we should test :D

That's what triggered this PR :)